### DISCUSSION TITLE
artisan moonshine:reset-password

### DISCUSSION THREAD

**Discussion by ramir1** [moonshine-software/moonshine#1274] [Ideas]
Cделать сброс пароля из консоли (с указание эл.адреса)

**Comment by lee-to** [comment]
Не уверен что такое нужно в коробке! Разработчик может за несколько секунд сделать подобное решение или реализовать забыли пароль или просто сделать php artisan tinker -> MoonShineUser::update()

**Reply by ramir1** [reply]
зависит от того, на какую целевую аудиторию Вы этот проект делаете.

**Reply by ramir1** [reply]
... просто взял из чата вопрос...

**Reply by lee-to** [reply]
> ... просто взял из чата вопрос... 

Да я так и понял, но там в чате был даже не разработчик
